README.md

```diff
@@ -2,26 +2,39 @@
 
-A _credentials.json_ file needs to exist in the same directory containing an `"api-key"` property for the [mailgun](http://www.mailgun.com/) account managing the iojs.org MX servers.
 
 ## Updating
 
-The [update](./update) directory contains a simple node program that you run and provide a domain (`update/update.js iojs.org`) which will read the aliases, fetch the list of mail routes from Mailgun and update the routes to make sure they match the required aliases. The Mailgun API key for the given domain is required to make this work. Use with `--dry-run` if you are unsure of the current status and what may be changed with an update.
+### Responsibility
+
+Since access to the Mailgun API key is required, only members of the @nodejs/build-infra team have the permission to push code to the `master` branch. That was done in order to minimize the posibility of mismatches between the information in this repo, and the actual email routes that are set up.
+
+### Procedure
+
+The [update](./update) directory contains a node program which will read the aliases mapping file, fetch the list of mail routes from Mailgun and update the routes to make sure they match the required state. The program is run by passing it a domain name as an agument (`update/update.js iojs.org`).
+
+The Mailgun API key for the given domain is required. It can be found in the `admin_logins.md` file in the secrets repo (build/infra/), or direclty via the Rackspace API. The key should be stored as a file named `iojs.org/credentials.json` in the form: `{ "api-key": "key-abc..." }` within a code tree.
 
-Members of the Build Working Group "Infra" team (Rod Vagg, Johan Bergström, Michael Dawson and João Reis as of writing) can access the Mailgun API key via the Rackspace API or in the `admin_logins.md` file in the secrets repo (build/infra/). This key must be placed in a file named `iojs.org/credentials.json` in the form: `{ "api-key": "key-abc..." }`.
+The programs can be used with `--dry-run` to verify the current status and what will be changed by an update.
 
-## nodejs.org email aliases
 
-nodejs.org aliases are managed by the Linux Foundation.  In order to set one up:
+# nodejs.org
 
-* create an iojs.org alias and associate the intended recipients to it.
-* email 'helpdesk@rt.linuxfoundation.org' requesting that the nodejs.org point to the alias. In the email, CC a Node.js Foundation contact (Executive Director or other staff) and provide context as to why the alias is needed.
+`@nodejs.org` email adresses are managed by the Linux Foundation. In order to set one up:
+* Create an iojs.org alias with associated recipients as described above.
+* Send an email to 'helpdesk@rt.linuxfoundation.org' requesting that the wanted nodejs.org address by directed to the new alias. Please CC an OpenJS Foundation contact person and provide context as to why the alias is needed.
 
-## License & copyright
+#### License & copyright
```

Comment:
can you remove these extra `##`'s please @refack?

Reply:
done.